Taboola SDK Swift Sample/Taboola SDK Swift Sample/SDK Standard and JS/TaboolaSTDandJSCollectionView.swift

```diff
@@ -0,0 +1,162 @@
+    
+    fileprivate struct TaboolaSection {
+        let placement: String
+        let whichSection: Int
+        
+        static let feed = TaboolaSection(placement: "Feed without video", whichSection: 3)
+    }
+    
+    override func viewDidLoad() {
+        
+        // Creating the feed.
+        taboolaFeed = taboolaViewFeed()
+    }
+    
+    // Function that creates the feed view with the appropriate attributes.
+    func taboolaViewFeed() -> TaboolaView {
+        let taboolaView = TaboolaView(frame: CGRect(x: 0, y: 0, width: view.frame.size.width, height: 0))
+        taboolaView.delegate = self
+        taboolaView.mode = "thumbs-feed-01"
+        taboolaView.publisher = "sdk-tester"
+        taboolaView.pageType = "article"
+        taboolaView.pageUrl = "http://www.example.com"
+        taboolaView.placement = "Feed without video"
+        taboolaView.targetType = "mix"
+        taboolaView.setInterceptScroll(true)
+        taboolaView.setOptionalModeCommands(["useOnlineTemplate": true])
+        taboolaView.viewID = viewId
+        taboolaView.fetchContent()
+        return taboolaView
+    }
+    
+    // Reseting the widget
+    deinit {
+        taboolaFeed.reset()
+    }
+    
+    // Function that sets the right constraints for widget
+    func setTaboolaConstraints(taboolaCell:UICollectionViewCell, taboolaView:inout TaboolaView) {
+        taboolaView.translatesAutoresizingMaskIntoConstraints = false
+        let horizConstraints = NSLayoutConstraint.constraints(withVisualFormat: "H:|-0-[taboolaView]-0-|", options: [], metrics: nil, views: ["taboolaView":taboolaView])
+        let vertConstraints = NSLayoutConstraint.constraints(withVisualFormat: "V:|-0-[taboolaView]-0-|", options: [], metrics: nil, views: ["taboolaView":taboolaView])
+        taboolaCell.addConstraints(horizConstraints)
+        taboolaCell.addConstraints(vertConstraints)
+    }
+}
+
+extension TaboolaSTDandJSCollectionView: UICollectionViewDataSource, UICollectionViewDelegateFlowLayout {
+    
+    func numberOfSections(in collectionView: UICollectionView) -> Int {
+        return 4
+    }
+    
+    // Choosing how many items(cells) to be in each sections:
+    func collectionView(_ collectionView: UICollectionView, numberOfItemsInSection section: Int) -> Int {
+        if (section == 0)
+        {
+            return 0
+        }
+        else if (section == 2)
+        {
+            return 3
+        }
+        return 1
+    }
+    
+    // Function that creates each cell with the info required.
+    func collectionView(_ collectionView: UICollectionView, cellForItemAt indexPath: IndexPath) -> UICollectionViewCell {
+        // Creating identifiers for the cellviews.
+        let taboolaIdentifier = "TaboolaCell"
+        let taboolaJSIdentifier = "TaboolaJSCell"
+        let normalIdentifier = "normalCell"
+        
+        // Switch case for each section.
+        switch indexPath.section {
+            
+        case 1:
```

Comment:
Please use enum like you used in case 'TaboolaSection.feed.whichSection'

Reply:
fixing